src/nuxeo-cloud/what-is-nuxeo-cloud.md

```diff
@@ -0,0 +1,208 @@
+---
+title: What's Nuxeo Cloud
+review:
+    comment: ''
+    date: '19-03-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 100
+toc: true
+---
+
+Nuxeo Cloud is a fully hosted and managed offering providing our customers with setup, support and maintenance for a complete Nuxeo implementation.
+
+This includes support and management of the infrastructure, third-party services, and application, including all deployments and/or changes to production and pre-production environments.
+
+![]({{file name='cloud-architecture.png'}})
+
+## Nuxeo Cloud Environments
+
+The Nuxeo Cloud Operations (NCO) team provides different environments as part of a standard configuration with the latest LTS and Hotfixes applied.
+
+### Production
+
+Primary environment used by customers:
+
+- Full stack deployment of Nuxeo and all required services
+- Complete monitoring and support
+- PCI/DSS and SOC2 Certified (as of 2019)
+
+### Pre-Production
+
+Environment used to validate deployments before going to production. These deployments may include cloud platform changes, hotfixes and security patches:
```

Comment:
change this to 

Pre-Production environments are used to validate all deployments before moving to Production. Pre-Production environments are a scaled down version of Production and include:

- Support and management of all changes and deployments by the Nuxeo Cloud team
- Support for Production data including regular data updates
- PCI/DSS and SOC2 Certified (as of 2019)